kitchen.yml

```diff
@@ -0,0 +1,135 @@
+---
+driver:
+  name: docker
+
+driver_config:
```

Comment:
As @dafyddj mentioned in Slack, lines 4 and 5 here can be removed, so that we end up with:

```yaml
driver:
  name: docker
  use_sudo: false
  privileged: true
  provision_command: mkdir -p /run/sshd
  run_command: /lib/systemd/systemd
```

Reply:
The same is true for lower down under `platforms:` too.